apps/block_scout_web/lib/block_scout_web/api_router.ex

```diff
@@ -43,6 +43,7 @@ defmodule BlockScoutWeb.ApiRouter do
         "logs" => {RPC.LogsController, []},
         "token" => {RPC.TokenController, []},
         "stats" => {RPC.StatsController, []},
+        "makerdojo" => {RPC.MakerdojoController, []},
```

Comment:
I don't think you need to add it to the API router since you're not adding a new API endpoint.

Reply:
0x50B04CAA099008d9435DaDe12B313ADB6dFd06F2